backend/api/src/test/kotlin/software/shonk/adapters/incoming/ShorkInterpreterControllerV1IT.kt

```diff
@@ -57,4 +54,11 @@ class ShorkInterpreterControllerV1IT() : AbstractControllerTest() {
         assertEquals(HttpStatusCode.BadRequest, result.status)
         assert(result.bodyAsText().contains("No player with that name in the lobby"))
     }
+
+    @Test
+    fun `test create a new lobby`() = runTest {
```

Comment:
I would like to see tests covering the scenario of creating multiple lobbies, so we can ensure that lobbyIds are different for each lobby.